XFBindableStackLayout/Controls/BindableStackLayout.cs

```diff
@@ -1,55 +1,134 @@
         }
 
+        public event EventHandler<SelectedItemChangedEventArgs> SelectedItemChanged;
+
         public IEnumerable ItemsSource
         {
             get { return (IEnumerable)GetValue(ItemsSourceProperty); }
             set { SetValue(ItemsSourceProperty, value); }
         }
-        public static readonly BindableProperty ItemsSourceProperty =
-            BindableProperty.Create(nameof(ItemsSource), typeof(IEnumerable), typeof(BindableStackLayout),
-                                    propertyChanged: (bindable, oldValue, newValue) => ((BindableStackLayout)bindable).PopulateItems());
+        public static readonly BindableProperty ItemsSourceProperty = BindableProperty.Create(
+            nameof(ItemsSource),
+            typeof(IEnumerable),
+            typeof(BindableStackLayout),
+            propertyChanged: (bindable, oldValue, newValue) => ((BindableStackLayout)bindable).PopulateItems()
+        );
 
         public DataTemplate ItemDataTemplate
         {
             get { return (DataTemplate)GetValue(ItemDataTemplateProperty); }
             set { SetValue(ItemDataTemplateProperty, value); }
         }
-        public static readonly BindableProperty ItemDataTemplateProperty =
-            BindableProperty.Create(nameof(ItemDataTemplate), typeof(DataTemplate), typeof(BindableStackLayout));
+        public static readonly BindableProperty ItemDataTemplateProperty = BindableProperty.Create(
+            nameof(ItemDataTemplate), typeof(DataTemplate), typeof(BindableStackLayout)
+        );
 
         public string Title
         {
             get { return (string)GetValue(TitleProperty); }
             set { SetValue(TitleProperty, value); }
         }
-        public static readonly BindableProperty TitleProperty =
-            BindableProperty.Create(nameof(Title), typeof(string), typeof(BindableStackLayout),
-                                    propertyChanged: (bindable, oldValue, newValue) => ((BindableStackLayout)bindable).PopulateHeader());
+        public static readonly BindableProperty TitleProperty = BindableProperty.Create(
+            nameof(Title), typeof(string),
+            typeof(BindableStackLayout),
+            propertyChanged: (bindable, oldValue, newValue) => ((BindableStackLayout)bindable).PopulateHeader()
+        );
+
+        public object SelectedItem
+        {
+            get { return GetValue(SelectedItemProperty); }
+            set { SetValue(SelectedItemProperty, value); }
+        }
+
+        public static BindableProperty SelectedItemProperty = BindableProperty.Create(
+            propertyName: "SelectedItem",
+            returnType: typeof(object),
+            declaringType: typeof(BindableStackLayout),
+            defaultValue: null,
+            defaultBindingMode: BindingMode.OneWay,
+            propertyChanged: OnSelectedItemChanged
+        );
+
+        private static void OnSelectedItemChanged(BindableObject bindable, object oldValue, object newValue)
```

Comment:
In order to keep the code style consistent the access modifier `private` should be removed as it is a default access modifier.